CIP-1694/README.md

```diff
@@ -1110,7 +1086,7 @@ Votes from unregistered SPOs or DReps count as having zero stake.
 > To avoid confusion, we will only use the word 'Abstain' from this point onward to mean an on-chain vote to abstain.
 
 The governance credential witness will trigger the appropriate verifications in the ledger according to the existing `UTxOW` ledger rule
-(i.e. a signature check for verification keys, and a validator execution with a specific vote redeemer and new Plutus purpose for scripts).
+(i.e. a signature check for verification keys, and a validator execution with a specific vote redeemer and new Plutus script purpose for scripts).
```

Comment:
we need to mention that the changes to the context and the integrity hash will be decided on and fixed in the implementation process.

Reply:
rather, we should say that the decisions made are in this PR and if needed, will be detailed in an addendum or something?
https://github.com/cardano-foundation/CIPs/pull/576